drf_simple_apikey/permissions.py

```diff
@@ -19,3 +19,43 @@ def has_object_permission(
         self, request: HttpRequest, view: typing.Any, obj
     ) -> bool:
         return request.user.is_active
+
+
+class ReadOnlyEntity(BasePermission):
+
+    message = "API key is read only."
```

Comment:
I think we can change the text to "This resource" instead of "API key". 

For example, "This resource is read-only." I think it's fair to assume endpoints as resources here, as we are working with a REST API.